Makefile

```diff
@@ -76,6 +76,26 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
+### build rpm for apisix-openresty:
+.PHONY: package-apisix-openresty-centos7-rpm
+package-apisix-openresty-centos7-rpm:
+	./build-apisix-openresty-centos7.sh
```

Comment:
We should provide a docker build environment like the other "build-apisix-xxx" tasks.

Reply:
update